openfecli/commands/plan_rbfe_network.py

```diff
@@ -229,6 +229,7 @@ def plan_rbfe_network(
     if overwrite_charges:
         write("\tOverwriting partial charges")
     write("")
+    write(f"\t{n_protocol_repeats=} ({n_protocol_repeats} simulation repeat(s) per protocol unit)\n")
```

Comment:
```suggestion
    write(f"\t{n_protocol_repeats=} ({n_protocol_repeats} simulation repeat(s) per protocol transformation)\n")
```
The wording here is incorrect, it's multiple repeat units in a transformation, not multiple repeats in a unit.